code/modules/mob/living/carbon/human/life.dm

```diff
@@ -899,7 +894,7 @@
 
 /mob/living/carbon/human/handle_vision()

```

Comment:
I didn't wanna do a full vision refactor because I don't want it to become tremendous like #5540